apps/ex_nvr/lib/ex_nvr/remote_storages.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ExNVR.RemoteStorages do
+  @moduledoc """
+  Context to manipulate remote storages
+  """
+
+  require Logger
+
+  import Ecto.Query
+
+  alias ExNVR.Model.{RemoteStorage}
+  alias ExNVR.Repo
+
+  @spec create(map()) :: {:ok, RemoteStorage.t()} | {:error, Ecto.Changeset.t()}
+  def create(params) do
+    params
+    |> put_bucket_default_value()
```

Comment:
The `bucket` is mandatory, the region is the one having the default value: `us-east-1`.
Also set the default value on the `schema`.